README.rst

```diff
@@ -96,6 +96,7 @@ Available Features
 * CSV Export of monitoring data
 * An overview of the status of the network is shown in the admin dashboard,
   a chart shows the percentages of devices which are online, offline or having issues;
+  timeseries charts are shown for WiFi sessions and Traffic of all managed devices,
```

Comment:
there are also two timeseries charts which show respectively the total unique wifi clients and the the traffic flowing to the network.

I think we need to add a new brief section that gives a bit more info about this feature, shows a screenshot and explains that the traffic chart is configurable and needs to be adapted according to the name of the main interface the traffic flows through.